### DISCUSSION TITLE
StripTags option in Formatted Items display

### DISCUSSION THREAD

**Discussion by d1rOn** [directus/directus#16522] [Archive]
It would be great to add possibility of stripping HTML tags when displaying the field contents.
As for now, all fields with WYSIWYG interface displaying not so tidy.

**Comment by rijkvanzanten** [comment]
Use the formatted-value display! :) That's what it does ✨

**Reply by d1rOn** [reply]
Omg! How could I not notice...
Has it always worked like this?

**Reply by rijkvanzanten** [reply]
Yup! 😄

**Reply by d1rOn** [reply]
I'm ashamed 😳

**Reply by danielband** [reply]
> I'm ashamed 😳

Don´t be! I was using directus for over a year and told everybody using the app that it is not currently possible to strip the tags.. There are a lot of individual settings in directus, that are vital to adoption. But once they are set, that tool is just awesome!
